stage1/Solution.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
<p>Fail :(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akionakamura/the-cipher-challenge/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>